cs.html

```diff
@@ -1674,32 +1889,38 @@ <h2>Event summary</h2>
         <tr>
           <td><code>error</code></td>
           <td><code><a>ErrorEvent</a></code></td>
-          <td>The <code><a>QuicTransportBase</a></code> object has encountered an error.</td>
+          <td>The <code><a>DataTransport</a></code> object has encountered an error.</td>
         </tr>
         <tr>
           <td><code>statechange</code></td>
           <td><code><a>Event</a></code></td>
-          <td>The <code><a>QuicTransportState</a></code> changed.</td>
+          <td>The <code><a>DataTransportState</a></code> changed.</td>
         </tr>
         <tr>
           <td><dfn><code>receivestream</code></dfn></td>
           <td><code><a>ReceiveStreamEvent</a></code></td>
-          <td>A new <code><a>QuicReceiveStream</a></code> is dispatched to the
-          script in response to the remote peer creating a send only QUIC stream and
+          <td>A new <code><a>ReceiveStream</a></code> is dispatched to the
+          script in response to the remote peer creating a send-only stream and
           sending data on it. Prior to <code><a>receivestream</a></code>
-          firing, the <code><a>QuicReceiveStream</a></code> is added to
-          <code><a>QuicTransportBase</a></code>'s <a>[[\QuicTransportReadableStreams]]</a>
+          firing, the <code><a>ReceiveStream</a></code> is added to
+          <code><a>DataTransport</a></code>'s <a>[[\IncomingStreams]]</a>
           internal slot.</td>
         </tr>
         <tr>
           <td><dfn><code>bidirectionalstream</code></dfn></td>
           <td><code><a>BidirectionalStreamEvent</a></code></td>
-          <td>A new <code><a>QuicBidirectionalStream</a></code> is dispatched to the
-          script in response to the remote peer creating a bidirectional QUIC stream and
+          <td>A new <code><a>BidirectionalStream</a></code> is dispatched to the
+          script in response to the remote peer creating a bidirectional stream and
           sending data on it. Prior to <code><a>bidirectionalstream</a></code>
-          firing, the <code><a>QuicBidirectionalStream</a></code> is added to the
-          <code><a>QuicTransportBase</a></code>'s <a>[[\QuicTransportReadableStreams]]</a>
-          and <a>[[\QuicTransportWritableStreams]]</a> internal slots.</td>
+          firing, the <code><a>BidirectionalStream</a></code> is added to the
+          <code><a>DataTransport</a></code>'s <a>[[\IncomingStreams]]</a>
+          and <a>[[\OutgoingStreams]]</a> internal slots.</td>
+        </tr>
+        <tr>
+          <td><dfn><code>datagramsreceived</code></dfn></td>
```

Comment:
I think this is a duplicate of the event below (you might want to delete both of them though, since they aren't used anywhere currently).

Reply:
Yep, both removed.